framework_tvb/tvb/adapters/uploaders/crypting/crypting.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
This file is missing the tvb copyright header and author.
I think this is the demo on how the user should encrypt files locally before upload into TVB.
I think it needs a bit more text commenting the content (once it is done).
Should we maybe move it under `tvb.interfaces.command.demos.importers` ? Or maybe a notebook under tvb_documentation/demos ?
The public key file should sit very close to this example

Reply:
I added a notebook and placed it there.